iothub/service/src/Messaging/MessageFeedbackProcessorClient.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Subclient of <see cref="IotHubServiceClient"/> for handling cloud to device message feedback.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-c2d#message-feedback"/>.
+    public class MessageFeedbackProcessorClient
+    {
+        private readonly string _hostName;
+        private readonly IotHubConnectionProperties _credentialProvider;
+        private readonly IotHubConnection _connection;
+
+        /// <summary>
+        /// Get the <see cref="AmqpFeedbackReceiver"/> which can deliver acknowledgments for messages sent to a device/module from IoT hub.
```

Comment:
Don't link to the type. It is in the property definition and will already be included.